modules/data-wrangling/index.qmd

```diff
@@ -120,31 +91,63 @@ There are many helper functions that allow you to select variables by matching t
 * `starts_with()`: select all variables that start with a prefix.
 
 ```{r}
+#| eval: false
 penguins |>
     select(starts_with("bill"))
 ```
 
+```{r}
+#| echo: false
+penguins |>
+    select(starts_with("bill")) |>
+    head()
```

Comment:
Using `head()` changes the number of rows of the data in the output.
Is the default 10 rows too much? You can change that with `print(data, n=6)` if you must.

You can also set the default behaviour with:
```
options(tibble.print_min = 3L)
```